.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "cargo"
+    directory: "/" # Location of package manifests
+    open-pull-requests-limit: 1
```

Comment:
why do we need a limit?

Reply:
To not drown in PRs if dependabot suddenly wants to upgrade 100 different things. We could set the limit to, say, 3 but I kinda like only dealing with one PR at a time.

Reply:
Makes sense, thanks.